projects/pele-network/project.md

```diff
@@ -0,0 +1,11 @@
+---
+name: PELE Network
+description: "PELE Network is a DAO facilitating creative collaborations, offering tools for community incentivization, and keeping badge registries for reputation."
+logo: pele-logo.svg
```

Comment:
redundant `logo` key

Reply:
Sorry, but I don't understand what the issue is?